src/views/_proxied-dot-io/product-docs-landing/index.tsx

```diff
@@ -2,20 +2,22 @@ import { ReactElement } from 'react'
 import CardLink from 'components/card-link'
 import FeaturedCard from './components/featured-card'
 import UseCaseCard from './components/use-case-card'
+import { ProductDocsLandingProps } from './types'
 import s from './style.module.css'
-import VAULT_CONTENT from './vault-content.json'
```

Comment:
Content and theme are now passed into the view component as props.

Reply:
EDIT: theme is no longer passed; as Bryce mentioned in another comment it's not necessary to do so thanks to our `ProductMeta` context 💯